sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AadWebSecurityConfigurerAdapter.java

```diff
@@ -65,10 +67,22 @@ protected void configure(HttpSecurity http) throws Exception {
                     .and()
                 .and()
             .logout()
-                .logoutSuccessHandler(oidcLogoutSuccessHandler())
-                .and()
-            .addFilterAfter(new AadHandleConditionalAccessFilter(), OAuth2AuthorizationRequestRedirectFilter.class);
+                .logoutSuccessHandler(oidcLogoutSuccessHandler());
         // @formatter:off
+
+        Filter handleConditionalAccessFilter = handleConditionalAccessFilter();
+        if (handleConditionalAccessFilter != null) {
+            http.addFilterAfter(handleConditionalAccessFilter, OAuth2AuthorizationRequestRedirectFilter.class);
```

Comment:
It's weird that this feature is partly contained in the library.

Reply:
Yes, a little bit, maybe we will add back in the future.